includes/Tasks/SetupSingleTask.php

```diff
@@ -46,7 +47,18 @@ public function perform(): bool
         $static_page->found_on_id = 0;
         $static_page->save();
 
-        Setup_Task::add_additional_files_to_db( $this->options->get( 'additional_files' ) );
```

Comment:
Not sure if I'm drawing the right conclusions here, but I think that if we don't implement this filter, we completely forego the additional files that are configured in the CMS, correct?

I would expect an `array_merge` or something here, or otherwise a check to see whether `apply_filters` actually returned some filters. Otherwise we should still process `$this->options->get( 'additional_files' )`, right?